client/state/data-layer/wpcom/read/sites/posts/follow/index.js

```diff
@@ -13,15 +13,15 @@ import { translate } from 'i18n-calypso';
  */
 import { READER_CONVERSATION_FOLLOW } from 'state/action-types';
 import { http } from 'state/data-layer/wpcom-http/actions';
-import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { dispatchRequestEx } from 'state/data-layer/wpcom-http/utils';
 import { errorNotice, successNotice } from 'state/notices/actions';
 import { updateConversationFollowStatus } from 'state/reader/conversations/actions';
 import { bypassDataLayer } from 'state/data-layer/utils';
 import getReaderConversationFollowStatus from 'state/selectors/get-reader-conversation-follow-status';
 
 import { registerHandlers } from 'state/data-layer/handler-registry';
 
-export function requestConversationFollow( { dispatch, getState }, action ) {
+export const requestConversationFollow = action => ( dispatch, getState ) => {
 	const actionWithRevert = merge( {}, action, {
 		meta: {
 			previousState: getReaderConversationFollowStatus( getState(), {
```

Comment:
Similar opportunity as in https://github.com/Automattic/wp-calypso/pull/28006/files#r227279120 -- set `meta.previousState` in action creator.